functions/federated_aws_rp/config.py

```diff
@@ -0,0 +1,35 @@
+import os
+import urllib.parse
+
+from jinja2 import Template
+
+
+class Config:
+    pass
+
+
+CONFIG = Config()
```

Comment:
This all seems a bit randomly ordered. Maybe set it inside `__init__` and order the variables a bit?

Reply:
Fixed in af102baa0442a35670c755c6eb1f21f6dbc759c4